clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -162,6 +163,9 @@ private void stop(String errorMessage) {
         transport.stop();
         connectionState = HubConnectionState.DISCONNECTED;
         logger.log(LogLevel.Information, "HubConnection stopped.");
+        if (onClosedCallback != null){
+            onClosedCallback.invoke(new Exception(errorMessage));
```

Comment:
Now I'm just creating a new Exception her but eventually I'll pass in a HubException.

Reply:
Will you do that update in this PR? Seems simple enough, just needs the new class.

Reply:
Sure I can do that here